src/templates/en/2022/methodology.html

```diff
@@ -0,0 +1,484 @@
+
+    <section>
+      <h4 id="rework-utils"><a href="#rework-utils" class="anchor-link">Rework Utils</a></h4>
+
+      <p>
+        This year&#8217;s <a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a> chapter revisits many of the metrics introduced in 2020's CSS chapter, which was led by <a href="{{ url_for('contributors', year=2019, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a>. Lea wrote <a hreflang="en" href="https://github.com/LeaVerou/rework-utils">Rework Utils</a> to more easily extract insights from Rework CSS's output. Most of the stats you see in the CSS chapter continue to be powered by these scripts.
+      </p>
+    </section>
+
+    <section>
+        <h4 id="parsel"><a href="#parsel" class="anchor-link">Parsel</a></h4>
+
+        <p>
+            <a hreflang="en" href="https://projects.verou.me/parsel/">Parsel</a> is a CSS selector parser and specificity calculator, originally written by <a href="{{ url_for('chapter', year=2019, lang=lang, chapter='css') }}">2020 CSS</a> chapter lead <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a> and open sourced as a separate library. It is used extensively in all CSS metrics that relate to selectors and specificity.
+        </p>
+    </section>
+
+    <section>
+      <h2 id="process"><a href="#process" class="anchor-link">Analytical process</a></h2>
+
+      <p>
+        The Web Almanac took about a year to plan and execute with the coordination of more than a hundred <a href="{{ url_for('contributors', year=year, lang=lang) }}">contributors</a> from the web community. This section describes why we chose the chapters you see in the Web Almanac, how their metrics were queried, and how they were interpreted.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="planning"><a href="#planning" class="anchor-link">Planning</a></h3>
+
+      <p>
+        The {{ year }} Web Almanac kicked off in March {{ year }} with a <a href="https://twitter.com/HTTPArchive/status/1508506002383069192">call for contributors</a>. We initialized the project with all 22 chapters from previous years and the community suggested additional topics that became two new chapters this year: <a href="{{ url_for('chapter', year=year, lang=lang, chapter='interop') }}">Interoperability</a> and <a href="{{ url_for('chapter', year=year, lang=lang, chapter='sustainability') }}">Sustainability</a>.
+      </p>
+
+      <p>
+        As we <a href="../2019/methodology#brainstorming">stated in the inaugural year&#8217;s Methodology</a>:
+      </p>
+
+      <blockquote>
+        One explicit goal for future editions of the Web Almanac is to encourage even more inclusion of underrepresented and heterogeneous voices as authors and peer reviewers.
+      </blockquote>
+
+      <p>
+        To that end, this year we&#8217;ve continued our <a href="https://github.com/HTTPArchive/almanac.httparchive.org/discussions/2165">author selection process</a>:
```

Comment:
@rviscomi anything to change here (and the sections below)?

Reply:
LGTM